.github/workflows/ci.yml

```diff
@@ -68,6 +68,8 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: engineerd/setup-kind@v0.4.0
+        with:
+          version: "v0.9.0"
```

Comment:
is this because setup-kind uses an older version of kind, or that we have to downgrade?

Reply:
Yeah, it uses an older version, so this was purely to upgrade the version

Reply:
FYI: https://github.com/engineerd/setup-kind/pull/26